docs/sphinx_click_handlers.py

```diff
@@ -0,0 +1,9 @@
+def process_description(app, ctx, lines):
+    if ctx.command.name == "main":
+        del lines[:3]
+
+    print(lines)
```

Comment:
**question:** Does this need to be there? The sphinx-click documentation says that the content is modified _in-place_, so I don't think it needs to be printed as well.

Reply:
My bad, that was some debug code I forgot to remove from the commit.